crl-test/folder.tf

```diff
@@ -0,0 +1,6 @@
+# Folder used to contain projects created in CRL integration tests.
+# We create a separate folder to scope the CRL test SA broad folder permissions to a single purpose folder.
+resource "google_folder" "test_resource_container" {
```

Comment:
```suggestion
Because the provider is [aliased](https://github.com/broadinstitute/terraform-ap-modules/blob/cf26e51445d88258624a6581071084d2bceac9c9/crl-test/provider.tf#L2), it needs to be explicitly specified in all resources to avoid permissions mix-ups.

resource "google_folder" "test_resource_container" {
  provider = google.target

```

Reply:
doh, thanks, done.